src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/SecretUtils.java

```diff
@@ -97,6 +101,18 @@ public static String base64DecodeToString(String s) {
         return null;
     }
 
+    /**
+     * Get the scope from a given {@code Secret}.
+     * @param s the secret whose scope we want to obtain.
+     * @return the scope for a given secret.
+     */
+    public static CredentialsScope getCredentialScope(Secret s) {
+        return CredentialsScope.valueOf(Optional
+                .ofNullable(s.getMetadata().getLabels().get(JENKINS_IO_CREDENTIALS_SCOPE_LABEL))
+                .orElse("global")
+                .toUpperCase());
```

Comment:
```suggestion
                .toUpperCase(Local.ROOT));
```

upper case is local dependant so we need to specify the LOCALE.  could be English but ROOT is as good as any.